doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,415 @@
+explicit and coordinated fashion.
+
+Option 1 conveys which trait sets may be managed, but not that some
+traits are unresolvable.
+
+Option 2 precludes the ability to differentiate between unmanaged trait
+sets and unresolvable component traits and so is not a viable option.
+
+## Further notes
+
+The chosen option effectively formalises the meaning of related API
+methods as follows:
+
+### `managementPolicy`
+
+For each Trait Set supplied by the Host (defining a concrete Entity type
+specialisation), the Manager will return a `TraitsData` instance imbued
+with:
+
+- Domain-specific policy traits that describe the Manager's handling of
+  entities with that trait set.
+- Any traits from the query set that have properties whose data can be
+  resolved by (read context) or registered with (write context) the
+  Manager when the supplied Entity Reference points to an entity of that
+  trait set.
+
+> Note:
+> Traits in the queried trait set with no properties will never be imbued
+> in the result. This significantly simplifies implementation in the
+> manager as it only needs to handle traits that it knows how to process
+> property values for. If a host wishes to determine if any given entity
+> has a particular trait, the ([forthcoming](https://github.com/OpenAssetIO/OpenAssetIO/issues/31))
+> `entityTraits` method should be used.
+
+### `entityTraits`
+
+For each Entity Reference supplied by the Host, the Manager will return
+a TraitSet containing the IDs of all traits possessed by that entity
+that define its type.
+
+This will include all traits required to correctly type that entity,
+regardless of whether the Manager is capable of provided data for all of
+those traits. I.e the Trait Set the entity was registered with if
+created through the API.
+
+E.g. in the above scenario, the Manager would be expected to return `{
+'file', 'image', 'colorMangaed' }` (i.e. those of the
+`ImageSpecification`) for any images, even though it will only ever
+resolve/register `file` trait data.
+
+This is the differentiator between `entityTraits` and `managementPolicy`
+- this queries the type of a specific entity, the other its data policy
+for a specific type of entity.
+
+### `resolve`
+
+For each Entity Reference and the Trait Set supplied by the host, the
+Manager will return a `TraitsData` instance imbued with:
+
+- Any trait with associated property values that is understood by
+  the Manager that the Entity has data for.
+
+> Note:
+> This means the traits imbued in the result should match (or be a
+> subset of) the intersection of the requested trait set and those
+> imbued in the `managementPolicy` response when queried with the
+> entity's trait set. I.e, no traits here that weren't in the policy
+> response.
+
+### `preflight`
+
+For each Entity Reference and the Trait Set supplied by the Host, the
+Manager will:
+
+- Error if a required Trait is missing from the supplied trait set (i.e.
```

Comment:
[question] Is this just a general comment, or does this relate to the DR somehow?  I.e. can the host use `managementPolicy` to determine the required traits?

Reply:
Was just for completeness, whilst we were there. It felt with including given the change proposed here is all about increasing the quality of information provided about what traits are supported/etc.